tests/integration/frameworks/drupal/mock_module_handler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * This modifies interfaces from the Symfony project.
```

Comment:
I think this can be removed.

Reply:
Just [these lines](https://github.com/newrelic/newrelic-php-agent/pull/665/commits/a525a87a2ea80b7893bf5936c701da5a357610c3#diff-fd53095561a99ce887c17efac95713ffd1bd3becf0798604a0568cbcfe4cb334L6-L9) please, our copyright can stay.

Reply:
re-added in [be5283b](https://github.com/newrelic/newrelic-php-agent/pull/665/commits/be5283be89e0c37a17299972752d58b624357e3c)